### ISSUE TITLE
DF to decimal issue

### ISSUE THREAD

**Issue by Miltiadis-Kon** [facioquo/stock-indicators-python#397]
### What happened?

Decimal point gets removed after conversion from float64. 

### Code usage

```python
# Import the yfinance library
import yfinance as yf

# Function to get the ticker data
def get_ticker(stock_symbol, timespan, time):
    # Fetch data from yfinance
    ticker = yf.Ticker(stock_symbol)
    # Fetch historical data from yfinance
    data = ticker.history(period=timespan, interval=time)
    return data

# Example: Get the data for the stock symbol 'AAPL' for the last 5 days with 1 minute interval
data = get_ticker('AAPL', '1d', '1m')
print(data) 

# Suppose that you have dataframe like the below.
#             date    open    high     low   close     volume
# 0     2018-12-31  244.92  245.54  242.87  245.28  147031456
# 1     2018-12-28  244.94  246.73  241.87  243.15  155998912
# 2     2018-12-27  238.06  243.68  234.52  243.46  189794032
# ...          ...     ...     ...     ...     ...        ...

from stock_indicators import Quote


print(data.columns)

print(data.head())

def get_historical_quotes(data):
    quotes_list = [
        Quote(d,o,h,l,c,v) 
        for d,o,h,l,c,v 
        in zip(data.index, data['Open'], data['High'], data['Low'], data['Close'], data['Volume'])
    ]
    return quotes_list

# Example: Get the historical quotes for the data
quotes = get_historical_quotes(data)
```

### Log output

```bash
Index(['Open', 'High', 'Low', 'Close', 'Volume', 'Dividends', 'Stock Splits'], dtype='object')
                                 Open        High         Low       Close  \
Datetime                                                                    
2024-08-15 09:30:00-04:00  224.550003  224.869995  223.285004  223.820007   
2024-08-15 09:31:00-04:00  223.869995  224.369995  223.699997  224.205902  
 
                            Volume  Dividends  Stock Splits  
Datetime                                                     
2024-08-15 09:30:00-04:00  2014142        0.0           0.0 

Quote on 09:30:00 - O:2245500030517578, H:2248699951171875, L:22328500366210938, C:22382000732421875, V:2014142
```

**Comment by LeeDongGeon1996** [comment]
@Miltiadis-Kon Thanks for your detail reporting. Could you let me know  `locale` setting on your environment?

**Comment by Miltiadis-Kon** [comment]
> @Miltiadis-Kon Thanks for your detail reporting. Could you let me know `locale` setting on your environment? I think it's a locale problem. To make it sure, I'd like to know the setting. If it's right, there's the patched version released on test PyPI.
> 
> ```shell
> pip install -i https://test.pypi.org/simple/ stock-indicators==1.3.1.dev9
> ```

Installed the patched version and it worked. 
Thanks.

**Comment by LeeDongGeon1996** [comment]
The patch finally has been released in [v1.3.1](https://pypi.org/project/stock-indicators/1.3.1/).
